lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
I'd much rather err on the side of longer names keeping convention, so naming the module `digital_ocean_sshkeys`.